server/handlers.go

```diff
@@ -0,0 +1,113 @@
+package server
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"sync"
+
+	"github.com/ASeegull/SmartFridge/server/database"
+	"github.com/gorilla/websocket"
+	log "github.com/sirupsen/logrus"
+)
+
+var wg sync.WaitGroup
```

Comment:
Wait group have to be connected or to be available from `main()` to handle graceful shutdown.